tests_integration/__tests__/early-exit.js

```diff
@@ -17,6 +17,57 @@ test("show usage with --help", () => {
   expect(result.status).toEqual(0);
 });
 
+test("show detailed usage with --help trailing-comma", () => {
```

Comment:
Thoughts on iterating over all of the options:

```js
const options = require("../../src/cli-constant").detailedOptions;

options.forEach(option => {
  test(`show detailed usage with --help ${option.name}`, () => {
    // ...
  });
});
```

The advantage is that when we add a new option we're forced to look at how it will show in help.